src/docfx/build/toc/parser/MarkdownTocMarkup.cs

```diff
@@ -134,7 +139,10 @@ TableOfContentsItem GetItem(HeadingBlock block)
                     currentItem.Name = GetLiteral(linkInline);
                 }
 
-                currentItem.Name = currentItem.Name ?? GetLiteral(block.Inline);
+                if (currentItem.Name.Value is null)
```

Comment:
use `Or` operator here as well?

Reply:
👍 There is only one subtle difference between `Or` and `??`, that is `Or` will always evaluate right hand side parameter, while `??` will *NOT* evaluate right hand side if left hand side is not null. In this case, `GetLiteral` is a relatively complex operation, so I am skipping the call intentionally to make it faster.

Reply:
ok, maybe add a comment for `Or` operation method?